toloka_provider/hooks/toloka.py

```diff
@@ -31,9 +31,11 @@ def __init__(self, toloka_conn_id: str = default_conn_name, *args, **kwargs) ->
         self.client: Optional[TolokaClient] = None
 
         conn = self.get_connection(toloka_conn_id)
+        self.toloka_host = conn.host or None
         extras = conn.extra_dejson
         self.toloka_token = extras.get("extra__toloka__token") or None
-        self.toloka_env = extras.get("extra__toloka__environment") or 'production'
+        self.toloka_env = extras.get("extra__toloka__environment",
```

Comment:
We need to raise Exception if both url and env are provided because it's hard for user understand what will be used. You can see [how it is handled in toloka-kit](https://github.com/Toloka/toloka-kit/blob/main/src/client/__init__.py#L252)

Reply:
The idea was not duplicate validations from TolokaClient here. At the end of constructior it's called self.get_conn() inside it creates TolokaClient instance. If user passes both host and environment it will raise exception from TolokaKit

Reply:
```python
self.toloka_env = extras.get("extra__toloka__environment", 'production') if not self.toloka_host else None
```
So env will be None if url is not None and there will be no error in TolokaClient(). 

And change " to ' please by the way)